### ISSUE TITLE
Editing collision of buildings vs bullets

### ISSUE THREAD

**Issue by ProgSys** [Sandern/lambdawars#2]
If a unit is inside a _BaseGarrisonable_ building collision, then it can't shoot out of it, as the collision is blocking the hit registration. There needs to be a way to add a filter to be able to create a exception for this case and other cases in the future.

My suggestion would it to do it via building name:
```python
unit.activeweapon.SetCollisionException (<string building name>)
unit.activeweapon.ClearCollisionExceptions()
```

**Comment by Sandern** [comment]
Fixed in 71cebc3

**Comment by ProgSys** [comment]
This does not apply to the veteran bolts and the sniper's trail. The snipers can kill but you don't see the bullet trail.
